test/unit/app/workers/asset_manager_create_asset_worker_test.rb

```diff
@@ -45,7 +45,7 @@ class AssetManagerCreateAssetWorkerTest < ActiveSupport::TestCase
     attachment = FactoryBot.create(:file_attachment, attachable: consultation)
     attachment.attachment_data.attachable = consultation
 
-    Services.asset_manager.expects(:create_asset).with(has_entry(access_limited: [@user.uid])).returns(@asset_manager_response)
```

Comment:
It should be safe to remove the `@user` variable now:

https://github.com/alphagov/whitehall/blob/97108ac2e9ec43bb482fe25351150020a615451d/test/unit/app/workers/asset_manager_create_asset_worker_test.rb#L9